mkosi/config.py

```diff
@@ -5007,8 +5019,11 @@ def parse_config(
     if args.debug_sandbox:
         ARG_DEBUG_SANDBOX.set(args.debug_sandbox)
 
+    if args.rerun_build_scripts and not args.verb.needs_build():
+        die(f"--rerun-build-scripts cannot be used with '{args.verb}' command")
+
     if args.cmdline and not args.verb.supports_cmdline():
-        die(f"Arguments after verb are not supported for {args.verb}.")
+        die(f"Arguments after verb are not supported for '{args.verb}' command")
```

Comment:
This should either have an article
```suggestion
        die(f"Arguments after verb are not supported for the '{args.verb}' command")
```
or change order
```suggestion
        die(f"Arguments after verb are not supported for command '{args.verb}'")
```